packages/react-reconciler/src/ReactFiber.new.js

```diff
@@ -14,10 +14,10 @@ import type {
   ReactFundamentalComponent,
   ReactScope,
 } from 'shared/ReactTypes';
-import type {Fiber} from './ReactInternalTypes';
+import type {Fiber} from 'shared/ReactInternalTypes';
 import type {RootTag} from './ReactRootTags';
-import type {WorkTag} from './ReactWorkTags';
-import type {TypeOfMode} from './ReactTypeOfMode';
+import type {WorkTag} from 'react-reconciler/src/ReactWorkTags';
+import type {TypeOfMode} from 'react-reconciler/src/ReactTypeOfMode';
```

Comment:
What is the reasoning? These are now absolute, but the ones below are relative. Seems unnecesary?

Reply:
yup, I should have looked more closely at the diff after the File Move refactoring in CLion/IntelliJ. fixing them now, should dramatically lower the number of changes.